src/components/picker/nimble-picker.js

```diff
@@ -467,6 +467,7 @@ export default class NimblePicker extends React.PureComponent {
         exclude,
         recent,
         autoFocus,
+        notFoundImg
```

Comment:
I think we need to set the default for this, or we'll break compatibility. 
You can add a default in the `shared-props` file under `PickerDefaultProps`.

If someone's using the component, they should not need to specify this property just to keep using it. 

That said, I think we may want something more flexible than a URL here, that can be evolved as more features are needed. 
Adding a top level property (where I'm considering "top-level" to be "Picker", since that's the component most people will use) just being a `String` is pretty inflexible. 

Here's a hypothetical scenario: I want to show a little minigame when there's no search results instead of an image. To do that, I'd now need to add _another_ top level property, that controls some aspect of the "not found" behavior. Something like `notFoundMinigame`.

This component (emoji-mart) has made this (in my opinion) misstep a few times already. If you take a look at the properties that `Picker` accepts, it's a pretty long list, and there's very little grouping (`i18n` being the exception).
Adding a new property that tweaks something about the way emoji are rendered requires adding a whole new prop (again growing the list).

If we made the property an Object, we could more easily evolve the Component's API, without requiring anyone that does not care and just wants to use the component to notice. 

So, say we made this prop an Object like:

```jsx
<Picker props={{
  notFound: { imgSrc: Url },
}} />
```

adding the ability to make the `NotFound` component also handle minigames would become an addition to the `NotFound` component's documentation and props. It would not require changing any of the other components along the way (because the `notFound` prop is passed down to the `NotFound` component) and would require only updating the `NotFound` component's logic.